src/codeu/chat/client/core/View.java

```diff
@@ -204,4 +206,66 @@ public int getConversationUpdate(Uuid owner, String title) {
     return messages;
   }
 
+  @Override
+  public String retrieveAuthInfo(Uuid id) {
```

Comment:
name this getAuthInfo to make it consistent with other function names in the class